docs/README.md

```diff
@@ -2,137 +2,139 @@
-be installed by an app team using a
-[`StackInstall`][stack-install-docs]
-Kubernetes API kind or with the [`stack
-install`][crossplane-cli-usage] command that
-installs Stacks with permissions only within a single namespace.
-
-The [Crossplane Stacks Guide][stack-user-guide] covers how to deploy a portable 
+Guide][services-user-guide] and the [Crossplane Stacks Guide][stack-user-guide].
+
+### Application Stacks
+
+App Stacks depend on Infra Stacks like [stack-gcp][stack-gcp],
+[stack-aws][stack-aws], and [stack-azure][stack-azure] to provide managed
+services via the Kubernetes API.
+
+App Stacks are pre-built and published to the [Stacks registry][stack-registry]
+where they can be installed by an app team using a
+[`StackInstall`][stack-install-docs] Kubernetes API kind or with the [`stack
+install`][crossplane-cli-usage] command that installs Stacks with permissions
+only within a single namespace.
+
+The [Crossplane Stacks Guide][stack-user-guide] covers how to deploy a portable
 [Wordpress Stack][stack-wordpress] into multiple clouds using Infra Stacks such
-as [stack-gcp][stack-gcp], [stack-aws][stack-aws], and [stack-azure][stack-azure].
+as [stack-gcp][stack-gcp], [stack-aws][stack-aws], and
+[stack-azure][stack-azure].
 
-### Build your own Stack 
-The [Crossplane Developer Guide][stack-developer-guide] shows how to
-build and extend Infra Stacks and App Stacks. 
+### Build your own Stack
 
-## Crossplane Workloads 
-Crossplane includes an extensible workload scheduler that observes application 
-policies to select a suitable target cluster from a pool of available clusters. 
-The workload scheduler can be customized to consider a number of criteria including 
-capabilities, availability, reliability, cost, regions, and performance while 
-deploying workloads and their resources.
+The [Crossplane Developer Guide][stack-developer-guide] shows how to build and
+extend Infra Stacks and App Stacks.
 
-Complex workloads can be modeled as a
-[`KubernetesApplication`][k8s-app-design]
-resource as done by the portable [Wordpress Stack][stack-wordpress] which
-automates the lifecycle of a
-[`KubernetesApplication`][k8s-app-design]
-including provisioning and secure connectivity to a managed `MySQLInstance`.
-
-## Crossplane Clusters 
-Crossplane supports dynamic provisioning of managed
-Kubernetes clusters from a single control plane with consistent multi-cluster
-best-practice configuration and secure connectivity between target Kubernetes
-clusters and the managed services provisioned for applications. Managed
-Kubernetes clusters can be dynamically provisioned with a `KubernetesCluster`
+## Crossplane Workloads
+
+Crossplane includes an extensible workload scheduler that observes application
+policies to select a suitable target cluster from a pool of available clusters.
+The workload scheduler can be customized to consider a number of criteria
+including capabilities, availability, reliability, cost, regions, and
```

Comment:
For the first time I actually feel like this is `TRUE`. All of these features can be enabled by label-based scheduling. I do think this sentence makes it sound like this process is a little bit more intelligent than it is at the moment, but I feel better about it than I have in the past.